### PR TITLE
Added loopdesigner

### PR DESCRIPTION
**Pull request by gabrielevajente**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gabrielevajente** [comment]
I am willing to be listed here

**Comment by gabrielevajente** [comment]
@conda-forge-admin, please ping team @conda-forge/help-python ready for review

**Comment by gabrielevajente** [comment]
@conda-forge-admin, please ping @conda-forge/help-python ready for review

File: recipes/loopdesigner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package:
+  name: loopdesigner
+  version: 0.1.1
```
Comment (ocefpaf) [inline recipes/loopdesigner/meta.yaml:3]:
```suggestion
{% set name = "loopdesigner" %}
{% set version = "0.1.1" %}

package:
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipes/loopdesigner/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+source:
+  url: https://git.ligo.org/gabriele-vajente/loopdesigner/-/archive/0.1.1/loopdesigner-0.1.1.tar.gz
```
Comment (ocefpaf) [inline recipes/loopdesigner/meta.yaml:6]:
```suggestion
  url: https://git.ligo.org/gabriele-vajente/loopdesigner/-/archive/{{ version }}/loopdesigner-{{ version }}.tar.gz
```

File: recipes/loopdesigner/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: 92ef172d2a990af29576f968e59388e5b8e1de6f2c4f17fa0b9eea589d336f34
+
+build:
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (ocefpaf) [inline recipes/loopdesigner/meta.yaml:10]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/loopdesigner/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/loopdesigner/meta.yaml:12]: I believe this one can be noarch.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

File: recipes/loopdesigner/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (ocefpaf) [inline recipes/loopdesigner/meta.yaml:17]:
```suggestion
    - python >=3.7
```

File: recipes/loopdesigner/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    - setuptools
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/loopdesigner/meta.yaml:20]:
```suggestion
    - python >=3.7
```

**Comment by gabrielevajente** [comment]
@ocefpaf Thnak you for your suggestions. I think I pushed all changes, let me know if there's anything else. Once we're good here, I'll do the same for the interactivefitting package

**Pull request merged**